src/components/VDataTable/index.js

```diff
@@ -5,7 +5,7 @@ import {
 import VDataTable from './VDataTable'
 import VEditDialog from './VEditDialog'
 
-const VTableOverflow = createSimpleFunctional('table__overflow')
+const VTableOverflow = createSimpleFunctional('v-table__overflow')
```

Comment:
This can be done after change the function for checking if the string already start with a `v-` and if true, don't double it. With that, we can progressively update the components without interfere other components that use this helper function.
https://github.com/vuetifyjs/vuetify/blob/b732a9ea02291d65fc603a6dd0bc5261d3f486a6/src/util/helpers.js#L1-L14

[List of dependent components and files](https://github.com/vuetifyjs/vuetify/search?utf8=%E2%9C%93&q=createSimpleFunctional&type=)